auction-house/program/src/state.rs

```diff
@@ -0,0 +1,42 @@
+use anchor_lang::{prelude::*, AnchorDeserialize, AnchorSerialize};
+
+use crate::constants::*;
+
+#[account]
+pub struct AuctionHouse {
+    pub auction_house_fee_account: Pubkey,
+    pub auction_house_treasury: Pubkey,
+    pub treasury_withdrawal_destination: Pubkey,
+    pub fee_withdrawal_destination: Pubkey,
+    pub treasury_mint: Pubkey,
+    pub authority: Pubkey,
+    pub creator: Pubkey,
+    pub bump: u8,
+    pub treasury_bump: u8,
+    pub fee_payer_bump: u8,
+    pub seller_fee_basis_points: u16,
+    pub requires_sign_off: bool,
+    pub can_change_sale_price: bool,
+    pub escrow_payment_bump: u8,
+    pub has_auctioneer: bool,
+    pub auctioneer_pda_bump: u8,
+}
+
+#[account]
+pub struct Auctioneer {
```

Comment:
I'm going to see if there is a way to specify new `trade_states` in an `Auctioneer` instance, and pull them down. If not, we need to use `ListingConfig` for feature expansion. I'll try and look at this tonight.